core/include/byteorder.h

```diff
@@ -48,9 +48,9 @@ typedef union __attribute__((packed)) {
  *                 between different byte orders at compile time.
  */
 typedef union __attribute__((packed)) {
-    uint32_t    u32;    /**< 32 bit representation */
-    uint8_t      u8[4]; /**< 8 bit representation */
-    uint16_t    u16[2]; /**< 16 bit representation */
+    uint32_t u32;       /**< 32 bit representation */
+    uint8_t u8[4];      /**< 8 bit representation */
+    uint16_t u16[2];    /**< 16 bit representation */
```

Comment:
@miri64 here's an example of previously aligned changes.

Reply:
Mh, but is the alignment really critical here? This looks more like some ASCII-art quirk of the original author. Compared to e.g. https://github.com/RIOT-OS/RIOT/pull/10867/files#diff-f4f930b62cff9df936337e8842e6c5d6R34 this doesn't really improve readability doesn't makes clearer what belongs where.

Reply:
ok, then let's uncrustify be our new ASCII artist.